doc/pachctl/pachctl_create-pipeline.md

```diff
@@ -102,66 +112,47 @@ Partition unit specifies the granularity at which input data is parallelized acr
-
-|                | Block |  Top-level Objects |  Repo  |
-|----------------|-------|--------------------|--------|
-|   Incremental  |  map  |                    |        |
-| Nonincremental |       |       reduce       | global |
-
-#### Defaults
-If no method is specified, the `map` method (Block + Incremental) is used by default.
+* If the input is non-incremental("NONE"), every job sees all the data. The first job sees file `/foo` and the second job sees file `/foo` and file `/bar`.
 
+* "File" (Top-level objects) means that if any part in a file (or alternatively any file within a directory) changes, then show all the data in that file (directory). For example, you may have vendor data files in separate directories by state -- the California directory contains a file for each california vendor, etc.  `Incremental: "file"` would mean that your job will see the entire directory if at least one file in that directory has changed. If only one vendor file in the whole repo was was changed and it was in the Colorado directory, all Colorado vendor files would be present, but that's it. 
 
-### Multiple Inputs
+#### Combining Partition unit and Incrementality
 
-A pipeline is allowed to have multiple inputs.  The important thing to understand is what happens when a new commit comes into one of the input repos.  In short, a pipeline processes the **cross product** of its inputs.  We will use an example to illustrate.
+For convenience, we have defined aliases for the three most commonly used (and most familiar) input methods: "map", "reduce", and "global". 
 
-Consider a pipeline that has two input repos: `foo` and `bar`.  `foo` uses the `file/incremental` method and `bar` uses the `reduce` method.  Now let's say that the following events occur:
+* A "map" (BLOCK + DIFF), for example, can partition files at the block level and jobs only need to see the new data. 
 
-```
-1. PUT /file-1 in commit1 in foo -- no jobs triggered
-2. PUT /file-a in commit1 in bar -- triggers job1
-3. PUT /file-2 in commit2 in foo -- triggers job2
-4. PUT /file-b in commit2 in bar -- triggers job3
-```
+* "Reduce" (FILE + NONE) as it's typically seen in Hadoop, requires all parts of a file to be seen by the same container ("FILE") and your job needs to reprocess _all_ the data in the whole repo ("NONE"). 
 
-The first time the pipeline is triggered will be when the second event completes.  This is because we need data in both repos before we can run the pipeline.
+* "Global" (REPO + NONE), means that the entire repo needs to be seen by _every_ container. This is commonly used if you had a repo with just parameters, and every container needed to see all the parameter data and pull out the ones that are relevant to it. 
 
-Here is a breakdown of the files that each job sees:
+They are defined below:
 
 ```
-job1:
-    /pfs/foo/file-1
-    /pfs/bar/file-a
-
-job2:
-    /pfs/foo/file-2
-    /pfs/bar/file-a
-
-job3:
-    /pfs/foo/file-1
```

Comment:
It looks like what happened here is that we edited some auto-generated documentation. When I regenerated the docs, the added comments got squashed. Let me try to add these to the source comments